src/Workspaces/Core/Portable/Storage/SQLite/v2/SQLitePersistentStorage_Helpers.cs

```diff
@@ -11,15 +11,6 @@ namespace Microsoft.CodeAnalysis.SQLite.v2
 {
     internal partial class SQLitePersistentStorage
     {
-        private static string GetProjectIdString(int projectPathId, int projectNameId)
-            => Invariant($"{projectPathId}-{projectNameId}");
-
-        private static string GetDocumentIdString(int projectId, int documentPathId, int documentNameId)
-            => Invariant($"{projectId}-{documentPathId}-{documentNameId}");
-
-        private static long CombineInt32ValuesToInt64(int v1, int v2)
-            => ((long)v1 << 32) | (long)v2;
```

Comment:
no need to manually have an encoding strategy to try to create compound primary keys out of other keys.